airflow/providers/mongo/sensors/mongo.py

```diff
@@ -28,9 +28,9 @@
 
 class MongoSensor(BaseSensorOperator):
     """
-    Checks for the existence of a document which
-    matches the given query in MongoDB. Example:
+    Checks for the existence of a document which matches the given query in MongoDB.
 
+    Example:
```

Comment:
```suggestion
    Checks for the existence of a document which matches the given query in MongoDB.

    Example:

	.. code-block:: pycon
```

`pycon` means Python console.